### PR TITLE
http://canihazip.com/ have changed into https://canihazip.com/ 

### PR DESCRIPTION
**Pull request by LarsBuur**
which makes the cli return 301 Moved Permanently. 
Have replaces http with https

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *
  * TODO: Implement HTTPS. - has to be implemented on canihazip.com first.
```
Comment (markogresak) [inline index.js:6]: This implements https, so the TODO comment is obsolete, please remove it.

**Review by markogresak — CHANGES_REQUESTED** [review]
Thanks for this fix! 👍 
Just one small change to keep the comment consistent with the code.

**Comment by markogresak** [comment]
I've merged this and I'll update the comment myself. 
There are some other fixes required to make the TravisCI build pass, this project is long overdue for some updates 😄

**Comment by LarsBuur** [comment]
Hi - thanks. Good job.

**Pull request merged**